test/plugin/test_out_google_cloud.rb

```diff
@@ -191,6 +191,20 @@ def test_parse_severity
     assert_equal('DEFAULT', test_obj.parse_severity('er'))
   end
 
+  # For non-grpc path, log name is reflected in the url, so we are stubbing the
+  # corresponding url containing the converted tag. To verify the tag is
+  # converted properly, we simply check the log entry is recorded successfully,
+  # and no exception (e.g. stubbing missing for certain url) is thrown.
+  def test_tag_acceptance
+    setup_logging_stubs_block = lambda do |converted_tag, setup_driver_block|
+      setup_logging_stubs([COMPUTE_PARAMS.merge(log_name: converted_tag)]) do
+        setup_driver_block.call
+      end
+    end
+    verify_log_name_block = ->(converted_tag) {}
```

Comment:
FWIW, if we modify `setup_logging_stubs` to add the appropriate `logName` field to each entry in @logs_sent, we should be able to reuse the log name verification code in both modes.